client/src/utils/quizManagement.ts

```diff
@@ -226,7 +225,31 @@ export const downloadUserQuizzesCsvQuery = async (
     // Create a temporary link element
     const link = document.createElement('a')
     link.href = URL.createObjectURL(blob)
-    link.download = 'userQuizzes.csv'
+    const currentDate = new Date()
+    const year = currentDate.getFullYear().toString().padStart(4, '0')
+    const month = (currentDate.getMonth() + 1).toString().padStart(2, '0')
+    const day = currentDate.getDate().toString().padStart(2, '0')
+    const hour = currentDate.getHours().toString().padStart(2, '0')
+    const minute = currentDate.getMinutes().toString().padStart(2, '0')
+    const second = currentDate.getSeconds().toString().padStart(2, '0')
+
+    const quizInfo = await getQuizInfoQuery(apollo, quizId)
+    const examName = quizInfo.name
+    link.download =
+      examName +
+      '_' +
+      year +
+      '-' +
+      month +
+      '-' +
+      day +
+      '-' +
+      hour +
+      '-' +
+      minute +
+      '-' +
+      second +
+      '.csv'
```

Comment:
Could be extracted into a util since its shared. No biggie though.

Reply:
I'll look in to it

Reply:
Done!